bundles/org.openhab.binding.heliosventiremote/src/main/resources/ESH-INF/i18n/heliosventiremote_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
remove if you don't provide information here.

Reply:
I just forgot about it. Giving German translations should be easy for me, is that enough to localize it and is there a tool the generate the file out of the XML? And is it good practice to translate?

Reply:
That one should also be done.